spec/unit/recipes/_captain_spec.rb

```diff
@@ -0,0 +1,74 @@
+# coding: UTF-8
```

Comment:
This file should just be captain_spec.rb right since the captain recipe doesn't start with an underscore.